rllib/examples/documentation/rllib_on_ray_readme.py

```diff
@@ -0,0 +1,45 @@
+# __quick_start_begin__
+import gym
+from ray.rllib.agents.ppo import PPOTrainer
+
+
+# Define your problem using python and openAI's gym API:
+class SimpleCorridor(gym.Env):
+    def __init__(self, config):
+        self.end_pos = config["corridor_length"]
+        self.cur_pos = 0
+        self.action_space = gym.spaces.Discrete(2)
+        self.observation_space = gym.spaces.Box(0.0, self.end_pos, shape=(1, ))
+
+    def reset(self):
+        self.cur_pos = 0
+        return [self.cur_pos]
+
+    def step(self, action):
+        if action == 0 and self.cur_pos > 0:
+            self.cur_pos -= 1
+        elif action == 1:
+            self.cur_pos += 1
+        # Set `done` flag when end of corridor (goal) reached.
+        done = self.cur_pos >= self.end_pos
+        # +1 when goal reached, otherwise -1.
+        reward = 1.0 if done else -0.1
+        return [self.cur_pos], reward, done, {}
+
+
+# Create an RLlib Trainer instance.
+trainer = PPOTrainer(
+    env=SimpleCorridor,
+    config={
+        "env": SimpleCorridor,
+        "env_config": {
+            "corridor_length": 20
+        },  # customize the environment
+        "num_workers": 3,  # parallelize environment rollouts
+    })
+
+# Train for n iterations.
+for i in range(5):
+    results = trainer.train()
+    print(f"Iter: {i}; avg. reward={results['episode_reward_mean']}")
+# __quick_start_end__
```

Comment:
We should add a short part right after, which is 1. how to get the model and 2. how to do inference.

This is the most basic scikit-learn example: https://scikit-learn.org/stable/auto_examples/classification/plot_digits_classification.html#sphx-glr-auto-examples-classification-plot-digits-classification-py

Note that they do the following:

```
# Create a classifier: a support vector classifier
clf = svm.SVC(gamma=0.001)

# Split data into 50% train and 50% test subsets
X_train, X_test, y_train, y_test = train_test_split(
    data, digits.target, test_size=0.5, shuffle=False
)

# Learn the digits on the train subset
clf.fit(X_train, y_train)

# Predict the value of the digit on the test subset
predicted = clf.predict(X_test)
```

^ the prediction aspect is very important!

Reply:
ah I just realized this is very similar to what you have on the RLlib docs readme already :) we should do the same here!

Reply:
Yeah, great point. I'll add another mini-script to illustrate inference after traininer.

Reply:
Done. To both ray README as well as RLlib README.